content/posts/bitcoin.md

```diff
@@ -170,6 +171,7 @@ the security of the Bitcoin network via Proof-of-Work (PoW) mining.
 Proof-of-Work entails a specific kind of cryptographic computation resembling
 a numerical puzzle that is difficult to solve but easy to verify. If
 computing power were evenly distributed throughout humanity (which of course
+<!-- why "could never be"? -->
 it's not and could never be), Bitcoin's Proof-of-Work mining would be a sort of Universal Basic Job, 
```

Comment:
I think "it's not" probably suffices, and you don't lose people who think it is possible to have an equitable distribution of resources.